domain/pool/poolstore.go

```diff
@@ -48,7 +48,23 @@ type storeImpl struct {
 //GetResourcePools Get a list of all the resource pools
 func (ps *storeImpl) GetResourcePools(ctx datastore.Context) ([]ResourcePool, error) {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("PoolStore.GetResourcePools"))
-	return query(ctx, "_exists_:ID")
+	req := map[string]interface{}{
+		"query": map[string]interface{}{
+			"bool": map[string]interface{}{
+				"must": map[string]interface{}{
+					"exists": map[string]interface{}{
+						"field": "ID",
+					},
+				},
+				"filter": map[string]interface{}{
```

Comment:
why do some use term for these exists queries and some use filter?

Reply:
correct, I'll fix it

Reply:
fixed